diff --git a/scripts/gutsy b/scripts/gutsy
index 90e1d5d..44f646e 100644
--- a/scripts/gutsy
+++ b/scripts/gutsy
@@ -1,11 +1,11 @@
 case $ARCH in
   amd64|i386)
-	default_mirror http://archive.ubuntu.com/ubuntu
+	default_mirror http://us-east-1.ec2.archive.ubuntu.com/ubuntu
 	;;
   sparc)
 	case $SUITE in
 	  gutsy)
-	default_mirror http://archive.ubuntu.com/ubuntu
+	default_mirror http://us-east-1.ec2.archive.ubuntu.com/ubuntu
 	;;
 	  *)
 	default_mirror http://ports.ubuntu.com/ubuntu-ports
